tests/examples/test_examples.py

```diff
@@ -56,6 +56,14 @@ def replace_mlflow_with_dev_version(yml_path):
         f.write(new_src)
 
 
+@pytest.fixture(scope="function", autouse=True)
+def clean_envs_and_cache():
+    yield
+
+    if get_free_disk_space() < 7.0:  # unit: GiB
+        process.exec_cmd(["./dev/remove-conda-envs.sh"])
+
+
```

Comment:
`./dev/remove-conda-envs.sh` removes all existing conda envs, then cleans up unused packages cache.

Reply:
Note that we could run `./dev/remove-conda-envs.sh` after each test execution (by removing `if get_free_disk_space() < 7.0:`), but it would drastically increase the total execution time of `test_examples.py`

Reply:
Any reason to choose 7 GB?

Reply:
because 7 GB is large enough to run the bert example?

```
tests/examples/test_examples.py::test_mlflow_run_example[pytorch/MNIST/example2-params17] PASSED | Free disk space: 11.9 GiB
tests/examples/test_examples.py::test_mlflow_run_example[pytorch/BertNewsClassification-params18] PASSED | Free disk space: 6.5 GiB
```

Reply:
I haven't tried 5 or 6 GB. They may also work.

Reply:
that's alright. Just wanted to know how you got the number :)